vega_sim/local_data_cache.py

```diff
@@ -453,14 +453,12 @@ def initialise_market_data(
     def initialise_transfer_monitoring(
         self,
     ):
-        base_transfers = []
-
-        base_transfers.extend(
-            data.list_transfers(data_client=self._trading_data_client)
-        )
+        base_transfers = data.list_transfers(data_client=self._trading_data_client)
 
         with self.transfers_lock:
             for t in base_transfers:
+                if t.status != events_protos.Transfer.Status.STATUS_REJECTED:
```

Comment:
Should this be == ?

Reply:
Or maybe gone entirely if it was for debugging

Reply:
Yes thanks! Should be `==`.

Thinking way back I think it was so we didn't fill the cache with rejected transfers we didn't care about. Part of me thinks we should scrap transfers from the cache entirely though but I want to do some profiling at some point anyway so will look at that then.